### PR TITLE
Page Notes stopwatch to tabs (15/15) completed

### PR DESCRIPTION
**Pull request by someshfengde**
I've completed page notes for 15 apps. 

@Coding-with-Adam  there are 3 apps remaining now (You haven't allocated those to anyone ig) 

- [ ] Text and annotations 
- [ ] Tick Formatting 
- [ ] Time series

### CODE REVIEW

File: examples/stopwatch.py [hunk 1]
```diff
@@ -35,7 +35,18 @@
...
+    
+    Inputs: 
+        n: number of intervals
+        input_value: number of seconds entered by user
+        start_watch: boolean value to start/stop the stopwatch
+        led_value: current value from LED display
 
+    Output:
+        led_value: current value from LED display
+    """
```
Comment (Coding-with-Adam) [inline examples/stopwatch.py:49]: Thanks for adding this Somesh, but for consistency purposes with all the other apps, we don't use a function description. I'll delete this file before accepting the PR.

Comment (Coding-with-Adam) [inline examples/stopwatch.py:49]:
```suggestion
```

**Review by Coding-with-Adam — COMMENTED** [review]
Some of these pages need a few changes. I'll merge the PR and make the changes in the main branch.

File: examples/stopwatch.py (hunk 1 above)
Comment (someshfengde) [reply examples/stopwatch.py]: I think we do have function docstring for some functions

Comment (Coding-with-Adam) [reply examples/stopwatch.py]: Ok, if you see them, let me know and I'll remove the descriptions. 

Thanks Somesh

Comment (someshfengde) [reply examples/stopwatch.py]: sure

**Pull request merged**